### ISSUE TITLE
Reworked action sheet

### ISSUE THREAD

**Issue by nolanw** [Awful/Awful.app#506]
![](https://user-images.githubusercontent.com/177228/29041792-9e432dca-7b8a-11e7-82dd-c1bf7f6ced7f.jpg)

I think the idea is this would be used in place of long-press menus and also the action menu on the posts view.

**Comment by nolanw** [comment]
> I'll also go back to beating this drum of I would love switching to these list style action menus for user/thread actions. Just makes a lot more sense now that I'm a lot smarter.
> 
> ![](https://user-images.githubusercontent.com/177228/64575834-17b19500-d34c-11e9-84a9-bd20802d7e8a.png)
> — [The Dave](https://forums.somethingawful.com/showthread.php?threadid=3837546&pagenumber=56&perpage=40#post497951278)

**Comment by nolanw** [comment]
> Hey I made adjusted icons for this view type. Hopefully they spark inspiration to get this view mode coded in!
> 
> ![](https://user-images.githubusercontent.com/177228/64575859-2d26bf00-d34c-11e9-841d-eaa6527e50c6.png)
> — [spanky the dolphin](https://forums.somethingawful.com/showthread.php?noseen=0&threadid=3837546&perpage=40&pagenumber=57#post498152800)

**Comment by realModusOperandi** [comment]
Will iOS 13 provide a standard control for this? I know the share sheet is going to look like this, would be nice not to have to roll our own.

**Comment by nolanw** [comment]
@realModusOperandi not really. [Context menus](https://developer.apple.com/design/human-interface-guidelines/ios/controls/context-menus/) are a new thing in iOS 13 but afaik they require a preview.

Agree it would be nice. Though I don’t mind making one, it’s a fun little well-constrained project.
